pkgs/applications/science/logic/isabelle/sha1.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchhg }:
+
+stdenv.mkDerivation {
+  name = "isabelle-sha1";
```

Comment:
This should probably also use pname + version.

Reply:
AFAICS this component has no concept of version.